bedrock/security/templates/security/web-bug-bounty.html

```diff
@@ -27,6 +27,51 @@ <h1 itemprop="name" class="title-shadow-box">Web and Services Bug Bounty Program
       </ul>
       <p>If two or more people report the bug together, or working independently at approximately the same time, the reward will be divided between them.</p>
 
+      <h2>How To Submit Bugs</h2>
+    
+      <p></p>The sooner we can reproduce and fix the bug, the sooner we can validate it, fix it, and send your reward.</p> 
+
+      <p>We prefer to receive bug reports in English. If English is not your native language and you are not fluent,  Mozilla has employees and volunteers from all over the world who can help. 
+If necessary, please write bug reports in your native language rather than using automated translation software. Automated translation software tends to mangle technical descriptions in indecipherable way.</p>
+
+      <p>There are three main things you can provide which will help us to evaluate your submission quickly and pay a bounty sooner:</p>
+      <ol>
+      <li>What is the attack scenario?</li>
+      <li>What is the step-by-step exploit process?</li>
+      <li>What is the security impact?</li>
+      </ol>
+      
+    <h3>What is the attack scenario?</h3>
+    
+    <p>For the attack scenario, please describe how an attacker would use the bug you are submitting, any necessary conditions for it to work, and what the attacker would gain through successful attack. Please answer the following questions to the best of your ability:</p>
+        <li>What is the necessary position of the attacker? (Example: Logged in to x website, unauthenticated, etc.)</li>
```

Comment:
These list items need to be wrapped in either a `<ul>` or `<ol>`, depending on whether the order is significant.